package.json

```diff
@@ -39,6 +39,38 @@
     "@angular/compiler-cli": "^0.6.1",
     "@angular/platform-server": "^2.0.0",
     "@angular/tsc-wrapped": "^0.3.0",
+    "@types/async": "^2.0.31",
+    "@types/browser-sync": "^0.0.32",
+    "@types/chalk": "^0.4.27-alpha",
+    "@types/chokidar": "^1.4.28",
+    "@types/connect-livereload": "^0.5.29",
+    "@types/core-js": "^0.9.32",
+    "@types/express": "^4.0.32",
+    "@types/glob": "^5.0.29",
+    "@types/gulp": "^3.8.31",
+    "@types/gulp-load-plugins": "^0.0.27",
+    "@types/gulp-protractor": "^1.0.28",
+    "@types/gulp-sass": "^0.0.28",
+    "@types/gulp-shell": "^0.0.28",
+    "@types/gulp-util": "^3.0.28",
+    "@types/jasmine": "^2.2.33",
+    "@types/micromatch": "^2.3.28",
+    "@types/mime": "^0.0.28",
+    "@types/minimatch": "^2.0.28",
+    "@types/node": "^6.0.38",
+    "@types/orchestrator": "^0.0.29",
+    "@types/parse-glob": "^3.0.28",
+    "@types/protractor": "^1.5.18",
+    "@types/q": "^0.0.30",
+    "@types/rimraf": "^0.0.27",
+    "@types/run-sequence": "^0.0.27",
+    "@types/selenium-webdriver": "^2.44.28",
+    "@types/systemjs": "^0.19.30",
+    "@types/through2": "^0.0.28",
+    "@types/vinyl": "^1.1.28",
+    "@types/vinyl-source-stream": "^0.0.27",
+    "@types/yargs": "^0.0.30",
+    "@types/zone.js": "^0.0.26",
     "async": "^2.0.0",
```

Comment:
Would you update all the dependencies for which you have installed `@types` to the same version as the typings themselves? For instance: `async` to be `^2.0.31` instead of `^2.0.0`.